pkg/operator/operator.go

```diff
@@ -241,36 +222,17 @@ func (o *Operator) WithControllers(ctx context.Context, controllers ...controlle
 	return o
 }
 
-func (o *Operator) WithWebhooks(ctx context.Context, ctors ...knativeinjection.ControllerConstructor) *Operator {
-	if !options.FromContext(ctx).DisableWebhook {
-		o.webhooks = append(o.webhooks, ctors...)
-		lo.Must0(o.Manager.AddReadyzCheck("webhooks", webhooks.HealthProbe(ctx)))
-		lo.Must0(o.Manager.AddHealthzCheck("webhooks", webhooks.HealthProbe(ctx)))
-	}
-	return o
-}
-
-func (o *Operator) Start(ctx context.Context, cp cloudprovider.CloudProvider) {
+func (o *Operator) Start(ctx context.Context) {
 	wg := &sync.WaitGroup{}
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
 		lo.Must0(o.Manager.Start(ctx))
 	}()
-	if options.FromContext(ctx).DisableWebhook {
-		log.FromContext(ctx).Info("conversion webhooks are disabled")
-	} else {
-		wg.Add(1)
-		go func() {
-			defer wg.Done()
-			// Taking the first supported NodeClass to be the default NodeClass
-			gvk := lo.Map(cp.GetSupportedNodeClasses(), func(nc status.Object, _ int) schema.GroupVersionKind {
-				return object.GVK(nc)
-			})
-			ctx = injection.WithNodeClasses(ctx, gvk)
-			ctx = injection.WithClient(ctx, o.GetClient())
-			webhooks.Start(ctx, o.GetConfig(), o.webhooks...)
-		}()
-	}
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		ctx = injection.WithClient(ctx, o.GetClient())
```

Comment:
We should not need to inject the client into the context. This was only needed for the expireAfter conversion